Ryujinx.Graphics.Vulkan/AutoFlushCounter.cs

```diff
@@ -0,0 +1,78 @@
+﻿using System.Diagnostics;
+
+namespace Ryujinx.Graphics.Vulkan
+{
+    internal class AutoFlushCounter
+    {
+        /** How often to flush on framebuffer change. */
```

Comment:
```suggestion
        /* How often to flush on framebuffer change. */
```
Could also use `// ` comments which is what we usually use for single line comments.